lesson04/script.js

```diff
@@ -0,0 +1,19 @@
+const ul = document.getElementById('js-list');
+const listContents = [
+ {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+ {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const fragment = document.createDocumentFragment();
+for(const list of listContents){
```

Comment:
IMO
It might be a good idea to change listContents to lists.
`for(const list of listContents){`
↓
`for(const list of lists){`
(const 単数形 of 複数形)  might be easier to recognize.

Reply:
Thank you for your comment!
Thank you for telling me a good way.
I'll try!